shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -158,9 +158,17 @@
 
     // The {@code SemanticsNode} within Flutter that currently has the focus of Android's
     // accessibility system.
+    //
+    // This is null when a node embedded by the AccessibilityViewEmbedder has the focus.
     @Nullable
     private SemanticsNode accessibilityFocusedSemanticsNode;
 
+    // The virtual ID of the currently embedded focused node.
+    //
+    // This is the ID of a node generated by the AccessibilityViewEmbedder if an embedded node is focused,
+    // -1 otherwise.
+    private int embeddedFocusedNodeId;
```

Comment:
Can we just make this an `Integer` and use `null` for non-existence rather than a magic number?